code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -200,6 +200,24 @@
 	..()
 	verbs -= /mob/living/verb/pulled //no pulling people deep into the blob
 
+/mob/living/simple_animal/hostile/blob/blobbernaut/Life()
+	if(..())
+		if(!(locate(/obj/effect/blob) in range(2, src)))
+			adjustHealth(maxHealth*0.025) //take 2.5% maxhealth as damage when not near the blob
+			var/list/viewing = list()
+			for(var/mob/M in viewers(src))
+				if(M.client)
+					viewing += M.client
+			flick_overlay(image('icons/mob/blob.dmi', src, "nautdamage", MOB_LAYER+0.1), viewing, 8)
```

Comment:
This probably could use a message to the blobbernaut why it's dying but i guess they will learn.